coprocess_python.go

```diff
@@ -213,8 +213,8 @@ func (d *PythonDispatcher) Reload() {
 func (d *PythonDispatcher) HandleMiddlewareCache(b *apidef.BundleManifest, basePath string) {
 	go func() {
 		runtime.LockOSThread()
-		defer runtime.UnlockOSThread()
```

Comment:
is this on purpose? are you sure it's safe to remove?

Reply:
@mvdan I've updated it with a `func` that performs both operations:`C.free` and `runtime.UnlockOSThread`.